cloudflare_worker/src/lib.rs

```diff
@@ -38,31 +42,31 @@ pub fn get_last_error_message() -> JsValue {
 }
 
 #[wasm_bindgen(js_name=fetchOcspFromDigicert)]
-pub async fn fetch_ocsp_from_digicert(fetcher: Function) -> Uint8Array {
+pub async fn fetch_ocsp_from_digicert(fetcher: Function) -> Result<Uint8Array, JsValue> {
     let fetcher = Box::new(sxg_rs::fetcher::js_fetcher::JsFetcher::new(fetcher));
-    let request = WORKER.fetch_ocsp_from_digicert(fetcher).await;
-    Uint8Array::from(request.as_slice())
+    let request = get_worker()?.fetch_ocsp_from_digicert(fetcher).await;
+    Ok(Uint8Array::from(request.as_slice()))
 }
 
 #[wasm_bindgen(js_name=servePresetContent)]
-pub fn serve_preset_content(req_url: &str, ocsp_base64: &str) -> JsValue {
+pub fn serve_preset_content(req_url: &str, ocsp_base64: &str) -> Result<JsValue, JsValue> {
     let ocsp_der = ::base64::decode(ocsp_base64).unwrap();
-    if let Some(preset_content) = WORKER.serve_preset_content(req_url, &ocsp_der) {
-        JsValue::from_serde(&preset_content).unwrap()
+    if let Some(preset_content) = get_worker()?.serve_preset_content(req_url, &ocsp_der) {
+        Ok(JsValue::from_serde(&preset_content).unwrap())
     } else {
-        JsValue::UNDEFINED
+        Ok(JsValue::UNDEFINED)
     }
```

Comment:
Maybe rewrite as follows?

```
    Ok(get_worker()?.serve_preset_content(req_url, &ocsp_der).map_or(JsValue::UNDEFINED, |preset_content|
        JsValue::from_serde(&preset_content).unwrap()
    })
```

I'm on the fence. It's shorter, but I'm not sure it's more readable.

Reply:
Done